suite-native/assets/src/assetsSelectors.ts

```diff
@@ -0,0 +1,76 @@
+import { createSelector } from '@reduxjs/toolkit';
+import BigNumber from 'bignumber.js';
+
+import { Network, networksCompatibility, NetworkSymbol } from '@suite-common/wallet-config';
+import { selectAccounts, selectCoins } from '@suite-common/wallet-core';
+import { toFiatCurrency } from '@suite-common/wallet-utils';
+import { selectFiatCurrency } from '@suite-native/module-settings';
+
+type Assets = Partial<Record<NetworkSymbol, string[]>>;
+
+interface AssetType {
+    symbol: NetworkSymbol;
+    network: Network;
+    assetBalance: BigNumber;
+    fiatBalance: string;
+}
+
+export const selectAssetsFromGroupedAccounts = createSelector(
```

Comment:
I am still not happy about the name, in this phase it has nothing to do with assets and also you are not selecting **from** grouped accounts but you are selecting account balances grouped by network/coin so I would suggest `selectBalancesPerNetwork`

Also what about moving `reduce` from `selectAssetsWithBalances` to this selector and return already summed number? It should be easy and I don't think we will ever need it as standalone numbers.